srcpkgs/sunwait/template

```diff
@@ -1,8 +1,8 @@
 # Template file for 'sunwait'
 pkgname=sunwait
-reverts="0.20041208_1"
+reverts="0.20041208_2"
```

Comment:
should be added not replaced

Reply:
fixed

Reply:
in the same variable, space separated: `reverts="0.20041208_1 0.20041208_2"`

having 2 lines would just overwrite the first with the second